etl/steps/data/garden/who/2023-07-13/autopsy.meta.yml

```diff
@@ -0,0 +1,41 @@
+dataset:
+  title: European Health Information Gateway (WHO, 2022)
+  description: >-
+    Since the mid-1980s, Member States of the WHO European Region have been reporting essential health-related statistics
+    to the Health for All (HFA) family of databases, making it one of WHO’s oldest sources of data. As it is based on reported
+    data, rather than estimates, the HFA family of databases is also particularly valuable.
+
+
+    HFA databases bring together the indicators that are part of major monitoring frameworks relevant to the Region, such
+    as Health 2020 and the Sustainable Development Goals. The indicators cover basic demographics, health status, health determinants
+    and risk factors, as well as health care resources, expenditures and more.
+  licenses:
+  - name: CC BY-NC-SA 3.0 IGO
+    url: https://www.who.int/about/policies/publishing/copyright
+  sources:
+  - name: WHO (2022)
+    description: >-
```

Comment:
To avoid repeating text, you could use here:
`    description: *dataset-description >-`